timed/reports/views.py

```diff
@@ -253,6 +260,17 @@ def _create_workreport(self, from_date, to_date, today, project, reports,
         table[13 + len(reports), 2].formula = 'of:=SUM(B13:B{0})'.format(
             str(13 + len(reports) - 1))
 
+        pos = 18 + len(reports)
+        for task in tasks.items():
+            table.insert_rows(pos, 1)
+            table[pos, 0] = Cell(
+                '{0}:'.format(task[0]),
```

Comment:
with above change this could simply be task_name

Reply:
Done